tox.ini

```diff
@@ -1,39 +1,47 @@
 [tox]
-envlist = py27,py36,static,pidiff
+envlist = py27,py36
+skipsdist = True
 
 [testenv]
-deps=-rtest-requirements.txt
-commands=pytest -v {posargs}
-whitelist_externals=sh
-
-[testenv:py27]
-deps=
-  # Note: we need to explicitly list requirements.txt here
-  # so it's processed at the same time as the constraints file
-  -rrequirements.txt
-  -rtest-requirements.txt
+deps =
+    -rrequirements.txt
+    -rtest-requirements.txt
+commands =
+    python setup.py install
+    pytest -v {posargs}
+whitelist_externals = sh
 
 [testenv:static]
-deps=
-  pylint
-commands=
-  sh -c 'pylint-2 alt-src; test $(( $? & (1|2|4|32) )) = 0'
+deps =
+    -rtest-requirements.txt
+    black
+    pylint
+commands =
+    black --check alt-src
+    sh -c 'pylint tests/test_import/alt_src.py; test $(( $? & (1|2|4|32) )) = 0'
```

Comment:
I think you'll have to change this once alt_src.py is available in python module. Make sure to include this change in either of the PR.

Reply:
Yeah. Each of these alt-src PRs are independent and will need to be rebased/resolved depending on the order in which they're merged.